content/blog/2020-03-12-native-classroom/index.Rmarkdown

```diff
@@ -0,0 +1,93 @@
+
+To create a safe and engaging classroom structure, I take a few different abstract and concrete steps...
+
+In the abstract, I first understand that for many Native Americans, and other historically marginalized populations, community and storytelling are a central part of life, playing massive roles in individual identity. The culture of many classrooms does not always value communities and their stories in the same way... Western higher education was established in a highly individualistic culture, whose Protestant Christian roots emphasize dutiful and successful work (not necessarily community) as a core component of individual identity ^[[The Protestant Ethic and Higher Education in America](https://www.jstor.org/stable/pdf/1050005.pdf?refreqid=excelsior%3A2bea6bf49df970ca4232f691609d8547)]. This culture has been introduced to the world too often via painful and dehumanizing experiences, like colonization, conquest, genocide, and slavery ^[[Higher Education in the United States](https://education.stateuniversity.com/pages/2044/Higher-Education-in-United-States.html)]. So, the elevation of the individual through hustle and perfection, which is a common ambition of higher education, does not always feel familiar to students, and if familiar, it does not always feel safe. For many students, instead, a focus on community and connections to community makes classroom material much more accessible. 
+
+Second, I try to adjust my perspective as a teacher, understanding that
+
+- *I am operating on stolen land.*
+- *My students likely have an inseparable connection to their communities and their land.* 
+- *My students are strong and resilient in the face of intergenerational trauma.*^[[The National Native American Boarding School Healing Coalition - Impact of Historical Trauma](https://boardingschoolhealing.org/education/impact-of-historical-trauma/)]
+- *My own notions of health, medicine, and justice are highly individualistic and institutional.*
+
+Then concretely, I put the following principles into action.
+
+- *The destination is clear*: Roadmapping is central to effective teaching in any classroom. One of the biggest barriers for students are the tacit expectations teachers place on them, including why they are taking the class and their performance and behavior in it. Students should have a clear outline about what we're doing throughout and at each stage of the course: what they will learn from it, how and when their performance will be evaluated, and how it will be useful to them outside the classroom. Yup, a good written syllabus + explicit and regular expression of expectations and learning objectives are foundational for inclusive classroms. Doing this well takes time, practice, and planning. ^[[Learning Objectives and Backward Design](https://tep.uoregon.edu/learning-objectives-and-backward-design)]
```

Comment:
I would change the last two sentences to, "In other words, a good written syllabus and ... are foundational for inclusive classrooms." and include citation [9] without the last sentence.